src/components/custom-field-input-date/custom-field-date.css

```diff
@@ -12,3 +13,9 @@
   display: none;
   -webkit-appearance: none;
 }
+
+.calendar-container {
+  position: absolute;
+  background-color: var(--white);
+  z-index: 2147483647;
```

Comment:
Magic numbers on `z-index` always make me nervous. If there's a future where for instance we want to display a tooltip while hovering over the calendar, this might make it impossible (or at least very difficult, with a need to move it to a different z-stack).

I think it's fine for now, but we should probably have a map of `z-index` values somewhere in documentation and backfill for components like this.